distribution/docs/src/main/resources/_contents/_configuring/managing-features-contents.adoc

```diff
@@ -0,0 +1,13 @@
+
+=== Managing Features
```

Comment:
This may be the section we want to use to discuss optional feature such as the http proxy, etc. See my other comments.